### PR TITLE
PlatformCon 2024

### PR DESCRIPTION
**Pull request by rrrutledge**
Read [the proposal](https://github.com/rrrutledge/talks/blob/rrrutledge-patch-2/2024/PlatformCon/proposal.md).

### CODE REVIEW

File: 2024/PlatformCon/proposal.md [hunk 1]
```diff
@@ -1,19 +1,62 @@
 # Session title
 
+Apparating a Platform with InnerSource
```
Comment (jeffabailey) [inline 2024/PlatformCon/proposal.md:3]:
```suggestion
Operating a Platform with InnerSource
```

File: 2024/PlatformCon/proposal.md [hunk 2]
```diff
@@ -1,19 +1,62 @@
...
+* We become the de-facto company standard, running 88% of all web projects.
+* Our platform strategy spreads to other areas.
+* The InnerSource Commons supports the practice of InnerSource.
+* Lessons learned:
+  * Focus on outcomes.
+  * Build social relationships.
+  * Define contracts.
+  * Run like a startup.
+  * Communicate value.
+
```
Comment (jeffabailey) [inline 2024/PlatformCon/proposal.md:41]: Looks good, I wonder if this story could tack on "The Future" as a topic at the end mentioning what the ISC is, our partnership with Gartner, and the inevitable rise of InnerSource practices within internal business Platforms?

File: 2024/PlatformCon/proposal.md (hunk 1 above)
Comment (rrrutledge) [reply 2024/PlatformCon/proposal.md:3]: It's supposed to be a little funny, catchy title - so "apparate".

File: 2024/PlatformCon/proposal.md (hunk 2 above)
Comment (rrrutledge) [reply 2024/PlatformCon/proposal.md:41]: ✅

File: 2024/PlatformCon/proposal.md (hunk 1 above)
Comment (jeffabailey) [reply 2024/PlatformCon/proposal.md:3]: Oh, I looked up the word but didn't see a definition. :)

**Pull request merged**